ndk/src/event.rs

```diff
@@ -339,16 +339,16 @@ impl MotionEventFlags {
 }
 
 impl MotionEvent {
-    /// Constructs a MotionEvent from a pointer to a native `AInputEvent`
+    /// Constructs a MotionEvent from a pointer to a native [`ffi::AInputEvent`]
     ///
     /// By calling this method, you assert that the pointer is a valid, non-null pointer to a
-    /// native `AInputEvent`, and that that `AInputEvent` is an `AMotionEvent`.
+    /// native [`ffi::AInputEvent`] and that that [`ffi::AInputEvent`] is an `AMotionEvent`.
     #[inline]
     pub unsafe fn from_ptr(ptr: NonNull<ffi::AInputEvent>) -> Self {
```

Comment:
I don't think it's necessary to specify the pointer must be non-null in the documentation, that's already implied by `NonNull<>`... Guess it's a way of reading.